apps/crypto/src/App.jsx

```diff
@@ -34,67 +67,102 @@ const Element = styled.div`
+  function convertTimestamp(timestamp) {
+    let date = new Date(timestamp);
+    return date.toLocaleString();
+  }
+
+  for (let i = 0; i < values.values.length; i++) {
+    labels.push(timestampArray[i]);
+  }
+
+  const crypto_data = {
+    labels,
+    datasets: [
+      {
+        label: "Price",
+        data: priceArray,
+        borderColor: "rgb(255, 99, 132)",
+        backgroundColor: "rgba(255, 99, 132, 0.5)",
+      },
+    ],
   };
+
   return (
     <Container>
       <select value={name} onChange={handleChange}>
-        {values.map((e) => (
-          <option key={e.name} name={e.name}>
-            {e.name}
-          </option>
-        ))}
+        <option key={values.name} name={values.name}>
+          {values.name}
+        </option>
       </select>
-
-      {values.map((e) => {
-        if (e.name === name) {
-          return (
-            <Head key={e.name}>
-              <Body>
-                <Element>
-                  Name🪙 :<strong>{e.name}</strong>
-                </Element>
-                <Element>
-                  Symbol:<strong>{e.symbol}</strong>
-                </Element>
-              </Body>
-              <Element>Price💰:{e.metrics.market_data.price_usd}USD</Element>
-              <Body>
-                <Element>
-                  24Hr High📈 :{e.metrics.market_data.ohlcv_last_24_hour.high}
-                  USD
-                </Element>
-                <Element>
-                  24Hr Low📉 :{e.metrics.market_data.ohlcv_last_24_hour.low}
-                  USD
-                </Element>
-              </Body>
-              <Body>
-                <Element>
-                  24Hr Open :{e.metrics.market_data.ohlcv_last_24_hour.open}
-                  USD
-                </Element>
-                <Element>
-                  24Hr Close :{e.metrics.market_data.ohlcv_last_24_hour.close}
-                  USD
-                </Element>
-              </Body>
-            </Head>
-          );
-        }
-      })}
+      <Head>
+        <Body>
+          <Element>
+            Name :<strong>{values.name}</strong>
+          </Element>
+          <img src="https://img.icons8.com/color/24/000000/bitcoin--v1.png" />
+          <Element>
+            Symbol :<strong>{values.symbol}</strong>
+            {/* <img src="https://img.icons8.com/color/24/000000/bitcoin--v1.png" /> */}
+          </Element>
+        </Body>
+        <Body>
+          <Element>
+            Price :
+            <h1>
+              <strong>${values.values.at(-1)[1].toFixed(2)}</strong>
+            </h1>
+          </Element>
+        </Body>
+        <Body>
+          <Element>
+            <p>{convertTimestamp(values.values.at(-1)[0])}</p>
+          </Element>
+        </Body>
+        <Body>
+          <Element>
+            High :<strong>{values.values.at(-1)[2].toFixed(2)}</strong>
+          </Element>
+          <Element>
+            Low :<strong>{values.values.at(-1)[3].toFixed(2)}</strong>
```

Comment:
Not a crypto trader myself but maybe we shouldn't use `toFixed` here, a lot of times bigger coins like bitcoin vary after 2 decimal places. What do you think?

Reply:
I checked googles crypto and most were 2 decimals. but you are right some of them do. I'll check the ones messari has and change it. But still think It's cleaner to have the `toFixed` but maybe with more decimals